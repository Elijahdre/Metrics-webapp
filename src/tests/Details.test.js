import React from 'react';
import Renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import { Provider } from 'react-redux';
import store from '../redux/store';

import Details from '../components/details';

describe('Details', () => {
  it('renders', () => {
    const tree = Renderer.create(
      <Provider store={store}>
        <BrowserRouter>
          <Details />
        </BrowserRouter>
      </Provider>,
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
