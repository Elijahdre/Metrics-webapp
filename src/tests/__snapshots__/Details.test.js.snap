// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Details renders 1`] = `null`;

exports[`Details renders correctly 1`] = `null`;
